lib/api/controllers/collectionController.js

```diff
@@ -318,7 +326,7 @@ function validateSpecificationList (validator, list) {
       });
 
       // we resolve here because we do not always need to send an error each time this method is used
-      return Promise.resolve({valid: false, details: errors, description: 'Some errors with provided specifications.'});
+      return Bluebird.resolve({valid: false, details: errors, description: 'Some errors with provided specifications.'});
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 121 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)